### DISCUSSION TITLE
I have no idea what I am doing

### DISCUSSION THREAD

**Discussion by altrenaltren** [bitbybyte/fantiadl#124] [Q&A] [answered]
First off, let me apologize.

I have very little cs knowledge but I managed to download the python code into github desktop and can view it in Visual Studio code. The 1.8.5 executable in my downloads folder and the copy I made in github closes as soon as I open it. I tried disabling windows defender for the .exe and it still closes immediately. However clicking on the tab keeps it open; it closes as soon as I try to do anything, however. 

Thanks in advance for helping this poor soul

**Comment by bitbybyte** [comment, marked as answer]
https://github.com/bitbybyte/fantiadl/discussions/46

**Reply by altrenaltren** [reply]
thank you.

**Reply by altrenaltren** [reply]
How do you open the exe in powershell? it doesn't come up in the tab.
Also, I have powershell open and copy pasting the exe into it for the directory still causes the same problem of it closing.

**Reply by altrenaltren** [reply]
nevermind, I'm stupid.
